tests/testcase.coffee

```diff
@@ -25,11 +25,22 @@ class TestCase
     keyBindings = new KeyBindings @settings
     @keyhandler = new KeyHandler @view, keyBindings
     @register = @view.register
+    @name = options.name || "an anonymous test"
+    unless callback then throw "No callback in test"
+    if it?
```

Comment:
Actually thought I took `if it?` out. That was the intention, but I decided it would be simpler without that line.